sphinx/util/nodes.py

```diff
@@ -33,6 +34,49 @@
 caption_ref_re = explicit_title_re  # b/w compat alias
 
 
+class NodeMatcher(object):
+    """A helper class for Node.traverse().
```

Comment:
Descriptions about `Any` value of some attribute are necessary.

Reply:
Absolutely!